packages/nc-gui/composables/useGlobal/state.ts

```diff
@@ -82,6 +82,7 @@ export function useGlobalState(storageKey = 'nocodb-gui-v2'): State {
     firstUser: true,
     githubAuthEnabled: false,
     googleAuthEnabled: true,
+    oidcAuthName: '',
```

Comment:
Use `null` instead of an empty string as initial value, that way we don't need to check against empty strings in the template later on.